sdk/src/main/scala/com/horizen/account/state/WithdrawalRequest.scala

```diff
@@ -8,6 +9,8 @@ case class WithdrawalRequest(proposition: MCPublicKeyHashProposition, value: jav
   override type M = WithdrawalRequest
 
   override def serializer: ScorexSerializer[WithdrawalRequest] = WithdrawalRequestSerializer
+
+  def valueInZennies: Long = ZenWeiConverter.convertWeiToZennies(value)
```

Comment:
can be `val`, since the object itself and the result is immutable

Reply:
Done